src/platform/STM32/platform_mcu.h

```diff
@@ -360,52 +360,46 @@ extern uint8_t _dmaram_end__;
 #error Unknown MCU family
 #endif
 
-#if !defined(STM32G4) && !defined(STM32H7)
+#if defined(STM32F4) || defined(STM32F7)
 #define USE_TX_IRQ_HANDLER
 #endif
 
 #if defined(STM32H7)
-#define UART_TX_BUFFER_ATTRIBUTE DMA_RAM            // D2 SRAM
```

Comment:
This one is quite fine, but it may be changed for consistency. 
Only version with empty x_ATTRIBUTE and line comment is confusing (maybe only to me) - it is not obvious that empty attribute is used. The idea being `/* empty */` is to explicitly show that nothing is added to variable declaration.

Sorry, I'm overthinking it. I just like when code meaning is obvious at first glance, without even thinking about it (and reading all of it).

Reply:
I changed this one as putting a // comment at the end of a define results in the else or endif being coloured as an error by vscode. Using /* */ resolves this...